src/assets/styles/variables/mixin.scss

```diff
@@ -4,23 +4,6 @@
   $align-items: flex-start,
   $flex-wrap: nowrap
 ) {
-  @debug "why not working" #{$flex-direction};
-  @if ($flex-direction != row) {
-    flex-direction: $flex-direction;
-  }
-
-  @if ($justify-content != flex-start) {
-    justify-content: $justify-content;
-  }
-
-  @if ($align-items != flex-start) {
-    align-items: $align-items;
-  }
-
-  @if ($flex-wrap != nowrap) {
-    flex-wrap: $flex-wrap;
-  }
-
```

Comment:
Already noticed, but leave comment for future reference.

No point to set `if statement here` since you've already set default values.
default value will be applied if no value explicitly set when use this mixin, otherwise use values of each flex properties.

```
// apply all default values
@include flexbox;

// apply flex-direction: column, justify-content:center, and default to the rest
@include flexbox(column, center);

// apply align-items: center, and default to the rest
@include flexbox($align-items: center);

```